src/main/java/org/edumips64/client/Worker.java

```diff
@@ -0,0 +1,70 @@
+/* WebUI.java
+ *
+ * GWT facade for the EduMIPS64 core.
+ * (c) 2020 Andrea Spadaccini
+ *
+ * This file is part of the EduMIPS64 project, and is released under the GNU
+ * General Public License.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+package org.edumips64.client;
+
+import com.google.gwt.core.client.EntryPoint;
+import org.gwtproject.rpc.worker.client.worker.MessagePort;
+import org.gwtproject.rpc.worker.client.worker.MessageEvent;
+
+import jsinterop.annotations.JsType;
+
+import java.util.logging.Logger;
+
+@JsType(namespace = "jsedumips64")
```

Comment:
Unless you are running other JS inside this worker, this annotation is unnecessary (and may increase code size, possibly cause extra compile warnings).

Reply:
Done, thanks!